pages/en/lb4/Getting-started.md

```diff
@@ -9,298 +9,82 @@ summary: Write and run a LoopBack 4 "Hello World" project in JavaScript and Type
+extension with more features under development. CLI provides the fastest way to
+get started with a LoopBack 4 project that adheres to best practices.
 
-1. Create `app.js` file.
-1. Run the project.
-
-### Create `app.js`
-
-Create a file named `app.js`, and copy the following into it:
-
-```js
-const Application = require('@loopback/core').Application;
-
-const app = new Application();
-app.bind('message').to('Hello world!');
-app.get('message').then(value => {
-  console.log(value);
-});
-```
-
-{% include note.html content="It's common for the main JavaScript file to be called `index.js`,
-but in the next section when you create a [TypeScript file](#try-a-typescript-project), named
-`index.ts` that then gets compiled to `index.js`, which would overwrite the original file.
-So for this step, you're going to use `app.js` instead.
-" %}
-
-### Run the project
-
-To run your project, enter this command:
-
-```shell
-node app.js
+Install the CLI globally by running
+```sh
+npm i -g @loopback/cli
 ```
 
-You should see "Hello world!" written to the console.
-
-## Try a TypeScript project
-
-To try out a LoopBack 4 project in TypeScript, follow these steps:
-
-1. Create `tsconfig.json` file.
-1. Create `index.ts` file.
-1. Build the project.
-1. Run the project.
-
-### Create `tsconfig.json` file
-
-A `tsconfig.json` file in a directory indicates that it contains a TypeScript project, and specifies project files and  TypeScript compiler options. For more information, see the [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).
-
-Create a new file called `tsconfig.json` in the project root directory.  
-Edit this file and copy/paste the following JSON into it, depending on the version of Node.js you're using:
-
-**With Node 8.x**:
-
-```js
-{
-  "compilerOptions": {
-    "target": "es2017",
-    "module": "commonjs",
-    "strict": true
-  }
-}
-```
-
-**With Node 6.x**:
-
-```js
-{                                                                                              
-  "compilerOptions": {
-    "target": "es6",
-    "module": "commonjs",
-    "strict": true
-  }
-}
-```
-
-{% include note.html content="Be sure to set the `'compilerOptions.target'` property based on the version of Node you are using.
-" %}
-
-### Create `index.ts`
-
-Create a file named `index.ts` and copy the following code into it:
-
-```ts
-import {Application} from '@loopback/core';
-
-const app = new Application();
-app.bind('message').to('Hello world!');
-app.get('message').then(value => {
-  console.log(value);
-});
-```
-
-### Build the project
-
-To compile the TypeScript file to JavaScript, enter this command:
+## Create a new project
 
+The CLI tool will scaffold the project, configure TypeScript compiler and
+install all the required dependencies. To create a new project, run the CLI as
+follows and answer the prompts.
 ```
-./node_modules/typescript/bin/tsc
+lb4
```

Comment:
so it's no longer `lb4 app`? 
Referring to: https://github.com/strongloop/loopback-next/tree/master/packages/cli#usage

Reply:
I think by default it chooses an app project, but yeah being explicit here would be good, unless I missed some updates that changed how it worked.

Reply:
The default chosen is `app` when you run `lb4` but I've updated it to explicitly run that command.